hidapi/dummy.go

```diff
@@ -0,0 +1,4 @@
+//go:build windows
+// +build windows
+
+package hidapi
```

Comment:
```suggestion
// +build dummy

// Package c contains only a C file.
//
// This Go file is part of a workaround for `go mod vendor`.
// Please see the file dummy.go at the root of the module for more information.
package hidapi
```
And corresponding change for all of these packages